code/modules/1713/ores.dm

```diff
@@ -5,7 +5,7 @@
 	icon_state = "ore2"
 	w_class = ITEM_SIZE_SMALL
 	amount = 1
-	max_amount = 50
+	max_amount = 100
```

Comment:
I think it should be kept at 50 since the crafting of things has been balanced to the max stack being 50. The Ore Collector is for storing large quantities of ore, but theres also carts etc. that can assist with transport of ores.

Reply:
Ah alright got it, i'll remove when im available.